ipify/client_rest.go

```diff
@@ -51,6 +52,18 @@ func (client *clientRest) GetIp() (string, error) {
 	return ipResponse.IP, nil
 }
 
+func (client *clientRest) GetOutboundIP() (string, error) {
+	conn, err := net.Dial("udp", "8.8.8.8:53")
+	if err != nil {
+		return "", err
+	}
+	defer conn.Close()
```

Comment:
Shouldn't this be at the top before error handling? Is it possible, that the connection could be established with an error?:)

Reply:
yes